scripts/import/import_all_sv_data.pl

```diff
@@ -952,9 +972,16 @@ sub phenotype_feature {
   while (my $row = shift(@$rows)) {
     my $phenotype = $row->[0];
     next if ($phenotype eq '');
-    
+
+    if($phenotype eq 'NULL') {
+      $phenotype = undef;
+    }
+
     $phenotype =~ s/'/\\'/g;
-    $dbVar->do(qq{ INSERT INTO phenotype (description) VALUES ('$phenotype')});  
+
+    my $stmt_phenotype = $dbVar->prepare(qq[ INSERT INTO phenotype (description) VALUES (?) ]);
+    $stmt_phenotype->execute($phenotype || undef);
+
```

Comment:
Doesn't this allow the addition of null phenotype entries?

Reply:
It did. Changed it to skip empty or 'NULL' phenotypes.